main.py

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env python3
+"""
+release_monitor script entrypoint
+"""
+
+from release_monitor import config
+
+if __name__ == "__main__":
+    log = config.setup_logging()
+    raise NotImplementedError()
```

Comment:
Just to make sure we're on the same page, the idea is to get this PR merged before incorporating the logic from [lambda_function.py](https://github.com/SeisoLLC/release_monitor/blob/5cdddcca2b141878c91f60abb41f9185731f9719/lambda_function.py), right?  I am good with that.

Reply:
@JonZeolla You are correct.